### DISCUSSION TITLE
PR "check jobs" count capped at 100

### DISCUSSION THREAD

**Discussion by joshlf** [community/community#66366] [Mobile]
When displaying a PR, the iOS app caps the count of check jobs at 100. All jobs appear to be displayed in the list of jobs, but the count says “100” even if there are more.

I observed this issue with: https://github.com/google/zerocopy/pull/343

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.129.0 (108873295)
Device: iPhone 12 Pro (iOS 16.6)
</details>

**Comment by iskae** [comment]
Hi @joshlf 👋 

Thanks so much for reaching out ❤️ I can confirm that on Pull Request screen we are only showing `100 passed` even though there 50 more checks in that Pull Request. Fortunately, when we tap on `View all` we can show all 150 checks on the Pull Request 🎉 

I will reach out to our iOS colleagues so that we reflect the total number of checks properly in the summary section too 🙇
